motr/m0crate/scripts/gen_index_yaml_run_workload

```diff
@@ -0,0 +1,66 @@
+script_dir=`dirname $(readlink -f  $0)`
+M0CRATE=$script_dir/../m0crate
+
+if [ ! -f "$M0CRATE" ]; then
+	echo "$M0CRATE not exists."
+	exit
+fi
+
```

Comment:
Add detailed comment, describe what script tries to test.

Reply:
FIxed.